website_and_docs/data/sponsors.yml

```diff
@@ -87,6 +87,10 @@ bronze:
   # Set to false if no items are present
   enable: true
   item:
+    # Sponsorship renewal date: Apr 12, 2026
```

Comment:
This date should be 2025 and not 2026.

Reply:
Oh, in a different context, I thought Manoj was referring to the next renewal date.